packages/react-components/src/theming/icons.tsx

```diff
@@ -290,5 +291,7 @@ export const DEFAULT_COMPONENT_ICONS = {
   /* @conditional-compile-remove(vertical-gallery) */
   VerticalGalleryLeftButton: <ChevronLeft20Regular />,
   /* @conditional-compile-remove(vertical-gallery) */
-  VerticalGalleryRightButton: <ChevronRight20Regular />
+  VerticalGalleryRightButton: <ChevronRight20Regular />,
+  /* @conditional-compile-remove(video-background-effects) */
+  OptionsVideoBackgroundEffect: <VideoBackgroundEffect20Regular />
```

Comment:
Is this icon specifically for the button? or is this to be used in both places? we should have a icon for the button and the background blur so Contoso can override them independently in the composite and not effect the other.

Reply:
This is specific to the video effects button, will declare a separate icon once we implement blur, so that both can be controlled independently.